weasyl/commishinfo.py

```diff
@@ -463,6 +463,8 @@ def edit_content(userid, content):
 
 def remove_class(userid, classid):
     d.execute("DELETE FROM commishclass WHERE (classid, userid) = (%i, %i)", [d.get_int(classid), userid])
+    # poor man's cascade
+    d.execute("DELETE FROM commishprice WHERE (classid, userid) = (%i, %i)", [d.get_int(classid), userid])
```

Comment:
Its debatable whether this change belongs in this PR or not to be honest. There was an issue brought up in user acceptance that I couldn't replicate locally so i'd set this to go on testing as a bit of exploratory debugging.  
Essentially, if you remove a class without deleting its prices then create a new class, the old prices will be assigned to it. This can also cause other strange behaviour in rendering the commission prices portion of the user profile. So, I'd consider it at least unintended behaviour.    
While setting up prices is closely related to marketplace I think the scope of handling this change correctly as well as possibly the numerous other issues that could come up while attempting to modernize commission prices should probably justify shifting the commit into #188